OrbitGl/App.cpp

```diff
@@ -111,6 +111,52 @@ void OrbitApp::SetCommandLineArguments(const std::vector<std::string>& a_Args) {
   }
 }
 
+static bool IsTrue(bool* flag) { return *flag; }
```

Comment:
I would prefer the use of lambdas in `absl::Condition`, to avoid these very generic tiny static functions.